electron_app/package.json

```diff
@@ -9,6 +9,10 @@
     "auto-launch": "^5.0.1",
     "electron-window-state": "^4.1.0",
     "minimist": "^1.2.0",
-    "png-to-ico": "^1.0.2"
+    "os": "^0.1.1",
+    "os-locale": "^3.0.1",
+    "png-to-ico": "^1.0.2",
+    "semver": "^5.6.0",
+    "spellchecker": "^3.5.0"
```

Comment:
is Native so will likely demand changes to the Build System

Reply:
ok i'll edit the travis file with the changes to allow builds for the spellchecker package

Reply:
The travis build doesn't build the electron app last I checked

Reply:
ok so which build system are you referring to then? Kindly clarify so that I know how to support the building of the spellchecker package well.

Reply:
The build system propably means the Mac PC used for building all electron apps of Riot

Reply:
@MTRNord ahh ok, got it. I also have a mac and its building fine, so I anticipate any issues in other Mac pc's so long as xcode is installed.